docker/script/setup

```diff
@@ -0,0 +1,25 @@
+if [[ -z "$DOCKER" ]]; then
+  docker compose -f docker/development.yml -p alaveteli run app ./docker/script/setup
+  exit
+fi
+
+bundle install
+
+git clone https://github.com/mysociety/alavetelitheme.git lib/themes/alavetelitheme
```

Comment:
Also noting that the older version of this theme that was already checked out here caused the rails errors, so that should probably be checked in some way?